codewit/lib/shared/tag/src/lib/tag.ts

```diff
@@ -0,0 +1,12 @@
+class tag {
+  name: string = '';
+
+  // constructor(name: string) {
+  //   this.name = name;
+  // }
+
+  // TODO: Implement the add method, make sure there are no duplicates
+  async add(tagName: string): Promise<void> {}
```

Comment:
Think you forgot to finish this function

Reply:
Actually in my opinion we should delete this tag library as we don't have anything to share between the backend and frontend unlike the language library where we have a limited set of languages being shared across frontend and backend

Reply:
@kbuffardi what's your opinion on this?

Reply:
Since you bring it up, you're probably right that the frontend most likely won't need any of the functionality of the class. Ultimately, it can probably just be relegated to the backend to manage a fairly simple list of tags used on exercises and demos.